### PR TITLE
Bugfix/log event mapping

### PR DESCRIPTION
**Pull request by asaharn**

### CODE REVIEW

File: store/reader.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ func (r *kustoSpanReader) GetTrace(ctx context.Context, traceID model.TraceID) (
 				"ParamTraceID": kusto.ParamType{Type: types.String},
 			},
 		)).MustParameters(kusto.NewParameters().Must(kusto.QueryValues{"ParamTraceID": traceID.String()}))
-
+	r.logger.Warn("*****************Searching for traceid*****************" + traceID.String())
```
Comment (ag-ramachandran) [inline store/reader.go]: we can remove this

File: store/kustoSpan.go [hunk 2]
```diff
@@ -49,18 +56,48 @@ func transformKustoSpanToModelSpan(kustoSpan *kustoSpan, logger hclog.Logger) (*
...
+	var events []event
+	err = json.Unmarshal(kustoSpan.Logs.Value, &events)
+	if err != nil {
+		logger.Warn(fmt.Sprintf("Error de-serializing data %s. The TraceId is %s and the SpanId is %s ", kustoSpan.Logs.String(), kustoSpan.TraceID, kustoSpan.SpanID))
+		return nil, err
+	}
 	var logs []dbmodel.Log
-	/*
-	   err = json.Unmarshal(kustoSpan.Logs.Value, &logs)
-	   if err != nil {
```
Comment (ag-ramachandran) [inline store/kustoSpan.go:60]: we can removed this un-used line

File: store/kustoSpan.go [hunk 3]
```diff
@@ -49,18 +56,48 @@ func transformKustoSpanToModelSpan(kustoSpan *kustoSpan, logger hclog.Logger) (*
 			tags[key] = elementString
 		}
 	}
-
 	if err != nil {
 		return nil, err
 	}
 	// TODO - This needs to get fixed !! Logs need more work on - Events needs - TODO //
```
Comment (ag-ramachandran) [inline store/kustoSpan.go]: This fix takes care of this TODO and can be removed

File: config/kusto_config.go [hunk 4]
```diff
@@ -6,11 +6,12 @@ import (
...
-	ClientSecret string `json:"clientSecret"`
-	TenantID     string `json:"tenantId"`
-	Endpoint     string `json:"endpoint"`
-	Database     string `json:"database"`
+	ClientID       string `json:"clientId"`
+	ClientSecret   string `json:"clientSecret"`
+	TenantID       string `json:"tenantId"`
+	Endpoint       string `json:"endpoint"`
+	Database       string `json:"database"`
+	TraceTableName string `json:"traceTable"`
```
Comment (ag-ramachandran) [inline config/kusto_config.go]: Guess this is for the table config feature and not yet implemented.

File: store/kustoSpan.go (hunk 2 above)
Comment (asaharn) [reply store/kustoSpan.go:60]: This is removed piece of code I guess.

File: config/kusto_config.go (hunk 4 above)
Comment (asaharn) [reply config/kusto_config.go]: Sure will take this up in another branch

**Pull request merged**